testmanage.py

```diff
@@ -8,6 +8,11 @@
 
 from django.core.management import execute_from_command_line
 
+try:
+    from wagtail.test.settings import STATIC_ROOT, MEDIA_ROOT
+except ImportError:
```

Comment:
Issue: Here we're using the try except to implicitly work as a version check. Why not just check the version properly? It's not clear that this check is for 3.0 and in the future could break functionality. 
```
from wagtail import VERSION as WAGTAIL_VERSION

if WAGTAIL_VERSION >= (3, 0):
    from wagtail.tests.settings import STATIC_ROOT, MEDIA_ROOT
else:
     from wagtail.tests.settings import STATIC_ROOT, MEDIA_ROOT
```

Reply:
Fixed